streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -574,17 +574,6 @@ public long compactionReadaheadSize() {
         return dbOptions.compactionReadaheadSize();
     }
 
-    @Override
-    public Options setRandomAccessMaxBufferSize(final long randomAccessMaxBufferSize) {
-        dbOptions.setRandomAccessMaxBufferSize(randomAccessMaxBufferSize);
-        return this;
-    }
-
-    @Override
-    public long randomAccessMaxBufferSize() {
-        return dbOptions.randomAccessMaxBufferSize();
-    }
-
```

Comment:
Unfortunately, we cannot simply remove these methods since they are kind of part of the public API. Could you please deprecate the methods and log a warn message?
Could you add the link to the PR that removes this option in the warn message?

Reply:
I just realized I had pasted the wrong link to the RocksDB commit 🤦. The original commit was actually reverted and later re-submitted. Here's the correct one:
https://github.com/facebook/rocksdb/pull/13288/files#diff-1536c1746bb06ab2e37be82324b437afc6b0ee159394d0d2c804532515f5a70a
(I’ve also updated the link in the PR description.)

This option is deprecated in the C++ code but has been removed entirely from the Java library.